plugins/panels/model_evaluation/__init__.py

```
@@ -751,6 +836,278 @@
+        if not self.can_delete_scenario(ctx):
+            return ctx.ops.notify(
+                "You do not have permission to delete scenarios",
+                variant="error",
+            )
+        scenario_id = ctx.params.get("id", None)
+        eval_id = ctx.params.get("eval_id", None)
+        store = self.get_store(ctx)
+        scenarios = store.get("scenarios") or {}
+        scenarios[eval_id].pop(scenario_id, None)
+        store.set("scenarios", scenarios)
+        ctx.ops.notify(f"Scenario deleted successfully!", variant="success")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)